core/dispatcher/src/whisk/core/container/ContainerPool.scala

```diff
@@ -349,16 +392,26 @@ class ContainerPool(
         debug(this, s"$prefix: keyMap = ${keyMapToString()}")
     }
 
+    private def getNodejsImageName(): String = {
```

Comment:
Yes and no.  The problem is that I don't have a WhiskAction in hand because this is a warmup and, AFAIK, we don't have an action-agnostic WhiskAction.